{{ cookiecutter.project_slug }}/Docker/entrypoint.sh

```diff
@@ -3,7 +3,7 @@
 set -e
 
 function help_text() {
-  cat << 'END'
+  cat << 'END'                                                              
```

Comment:
If this is an extra space, we should remove it

Reply:
Fixed that